sdk/spring/scripts/ci_add_spring_boot_dependencies_management.py

```diff
@@ -0,0 +1,72 @@
+# Python version 3.4 or higher is required to run this script.
+#
+# This script works for ci.
+#
+# How to use:
+# 1. Change `SPRING_BOOT_DEPENDENCIES_VERSION` and 'SPRING_CLOUD_DEPENDENCIES_VERSION'
+#    to the latest version in this script manually.
+# 2. Then the ci will automatically run command
+#    `python .\sdk\spring\scripts\ci_add_spring_boot_dependencies_management.py`.
+#
+# The script must be run at the root of azure-sdk-for-java.
+
+import os
+import time
+
+
+SPRING_BOOT_DEPENDENCIES_VERSION = '2.6.1'
+SPRING_CLOUD_DEPENDENCIES_VERSION = '2021.0.0'
+
+def add_dependency_management(c1,c2):
+    for root, _, files in os.walk("./sdk/spring"):
+    # for root, _, files in os.walk("D:/java/azure-sdk-for-java/sdk/spring/"):
+        for file_name in files:
+            file_path = root + os.sep + file_name
+            if file_name.startswith('pom') and file_name.endswith('.xml'):
+                with open(file_path, 'r', encoding='utf-8') as f:
+                    content = f.read()
+                    pos1 = content.find('<dependencies>')
+                    pos2 = content.find('<dependencyManagement>')
+                    if pos2 != -1:
+                        print("processing:" + file_path)
+                        print("add dependency management...")
+                        content = content[:pos2+41] + c1 + content[pos2+41:]
+                        with open(file_path, 'r+', encoding='utf-8') as f:
+                            f.writelines(content)
+                    else:
+                        print("processing:" + file_path)
+                        print("add dependency management...")
+                        content = content[:pos1] + c2 + content[pos1:]
+                        with open(file_path, 'r+', encoding='utf-8') as f:
+                            f.writelines(content)
+
+def add_dependency_management_all():
+    cores = '\n        <groupId>org.springframework.boot</groupId>'
+    cores += '\n        <artifactId>spring-boot-dependencies</artifactId>'
+    cores += '\n        <version>'+SPRING_BOOT_DEPENDENCIES_VERSION+'</version><!-- version test for ci -->'
+    cores += '\n        <type>pom</type>'
+    cores += '\n        <scope>import</scope>'
+    cores += '\n      </dependency>\n'
+    cores += '\n      <dependency>'
+    cores += '\n        <groupId>org.springframework.cloud</groupId>'
+    cores += '\n        <artifactId>spring-cloud-dependencies</artifactId>'
+    cores += '\n        <version>'+SPRING_CLOUD_DEPENDENCIES_VERSION+'</version><!-- version test for ci -->'
```

Comment:
Use string format instead of concatenate string.

Example:
```
'org.springframework.boot:spring-boot-starter-parent;{}'.format(SPRING_BOOT_VERSION),
```